custom-instrumentation/background-transactions/index.js

```diff
@@ -6,89 +6,22 @@
 
-//example1-basic
-function basicBackgroundTransaction() {
-    return new Promise((resolve) => {
-        const transactionName = 'myBasicTransaction'
-
-        // `startBackgroundTransaction()` takes a name, group, and a handler function to
-        // execute. The group is optional. The last parameter is the function performing
-        // the work inside the transaction. Once the transaction starts, there are
-        // three ways to end it:
-        //
-        // 1) Call `transaction.end()`. The `transaction` can be received by calling
-        //    `newrelic.getTransaction()` first thing in the handler function. Then,
-        //    when you call `transaction.end()` timing will stop.
-        // 2) Return a promise. The transaction will end when the promise resolves or
-        //    rejects.
-        // 3) Do neither. If no promise is returned, and `getTransaction()` isn't
-        //    called, the transaction will end immediately after the handler returns.
-
-        // Here is an example for the first case.
-        newrelic.startBackgroundTransaction(transactionName, function handle() {
-            const transaction = newrelic.getTransaction()
-            doSomeWork(function cb() {
-                transaction.end()
-                resolve()
-            })
-        })
-
-        /*
-         * Function to simulate async work.
-         */
-        function doSomeWork(callback) {
-            setTimeout(function work() {
-                callback()
-            }, 500)
-        }
-    })
-
-}
-
-//example2-grouping
-function groupingBackgroundTransaction() {
-    return new Promise((resolve) => {
-        const transactionName = 'myGroupedTransaction'
-
-        // The second parameter to `startBackgroundTransaction` may be a group to
-        // organize related background transactions on APM. More on this can be found
-        // on our documentation website:
-        // https://docs.newrelic.com/docs/apm/applications-menu/monitoring/transactions-page#txn-type-dropdown
-        const groupName = 'myTransactionGroup'
-
-        newrelic.startBackgroundTransaction(transactionName, groupName, function handle() {
-            const transaction = newrelic.getTransaction()
-            doSomeWork(function cb() {
-                transaction.end()
-                resolve()
-            })
-        })
-
-        /*
-         * Function to simulate async work.
-         */
-        function doSomeWork(callback) {
-            setTimeout(function work() {
-                callback()
-            }, 500)
-        }
-    })
-}
-
-//example3-results
-function resultsBackgroundTransaction() {
-    const transactionName = 'myResultTransaction'
-    // The return value of the handle is passed back from `startBackgroundTransaction`.
-    return newrelic.startBackgroundTransaction(transactionName, function handle() {
-        return "done"
-    })
+/**
+ * Function to simulate async work with a callback.
+ * @param callback
```

Comment:
If we are adding jsdoc, it should be in the correct format:
```suggestion
 * @param {function} callback
```